packages/dx-grid-core/tsconfig.lint.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Have we able to keep old alignment rules?

Reply:
These rules are defined in tslint.json in repository root directory. They are as close as possible to eslint rules we had

Reply:
Okay, I see. But many of my alignments are broke. I try to find anything to fix it.